### DISCUSSION TITLE
Cannot update uno-check unless I remove a nuget packages source in nuget.config

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#9322] [General]
For my nuget packages I use the repository provided by my Visual Studio subscription.

My nuget.config looks like:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
    <add key="Infragistics (Local)" value="C:\Users\Public\Documents\Infragistics\NuGet\" />
    <add key="Microsoft Visual Studio Offline Packages" value="C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\" />
    <add key="mcNugets" value="https://avinent.pkgs.visualstudio.com/_packaging/mcNugets/nuget/v3/index.json" />
  </packageSources>
  <packageRestore>
    <add key="enabled" value="True" />
    <add key="automatic" value="True" />
  </packageRestore>
  <bindingRedirects>
    <add key="skip" value="False" />
  </bindingRedirects>
  <packageManagement>
    <add key="format" value="0" />
    <add key="disabled" value="False" />
  </packageManagement>
</configuration>
```

When I try to update the uno-check I get the next error:

>error NU1301: Unable to load the service index for source >https://avinent.pkgs.visualstudio.com/_packaging/mcNugets/nuget/v3/index.json.
>Tool 'uno.check' failed to update due to the following:
>The tool package could not be restored.
>Tool 'uno.check' failed to install. This failure may have been caused by:
>
>* You are attempting to install a preview release and did not use the --version option to specify the version.
>* A package by this name was found, but it was not a .NET tool.
>* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
>* You mistyped the name of the tool.
>
>For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool

And sometimes other similar messages when I use `dotnet new` to generate a new Uno application. Usually when it executes `dotnet restore`

Then I have no other option than edit nuget.config and remove or comment this line:

```
<!--
<add key="mcNugets" value="https://avinent.pkgs.visualstudio.com/_packaging/mcNugets/nuget/v3/index.json" />
-->
```

and when I have executed the command, edit it again and remove the comment.

Is there a way to avoid it?

**Comment by jeromelaban** [comment]
You can use the [`--ignore-failed-sources`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-update#options) parameter to get you going.

**Reply by mcNets** [reply]
Thanks
